source/documentation/github/create-project.md

```diff
@@ -0,0 +1,79 @@
+# Create a new project in GitHub
+
+In GitHub, you can use a repository to store and collaborate on all of your project's code.
+
+You can also use:
+
+- [pull requests](https://docs.github.com/en/pull-requests) to propose changes to your code and receive reviews from others
+- [issues](https://docs.github.com/en/issues) to plan and track your work
+
+### Create a new repository
+
+To create a new repository, you should:
+
+1. Navigate to the [MoJ Analytical Services](https://github.com/moj-analytical-services) GitHub organisation page.
+2. In the repositories section, select **New**.
+3. Configure the properties of your repository, in line with the information below.
+4. Select **Create repository**.
```

Comment:
Is it worth adding a note to use a template repository at this point, I've copied in our template to AP Org, it gives some useful notes in the README to help setup repo https://github.com/moj-analytical-services/template-repository

Reply:
Added in 473de87